### PR TITLE
feat: use contract with kuai

### PR DESCRIPTION
**Pull request by yanguoyu**

### CODE REVIEW

File: packages/samples/mvp-dapp/src/actors/record.model.ts [hunk 1]
```diff
@@ -42,8 +50,10 @@ export type StoreType = {
  */
 @ActorProvider({ ref: { name: 'record', path: '/:codeHash/:hashType/:args/' } })
 @LockPattern()
+@TypePattern()
 @DataPrefixPattern(DAPP_DATA_PREFIX)
```
Comment (Keith-CY) [inline packages/samples/mvp-dapp/src/actors/record.model.ts:54]: Is the `DataPrefixPattern` redundant now?

Comment (yanguoyu) [reply packages/samples/mvp-dapp/src/actors/record.model.ts:54]: It uses to filter cell's data that start with `DAPP_DATA_PREFIX`

Comment (Keith-CY) [reply packages/samples/mvp-dapp/src/actors/record.model.ts:54]: But `TypePattern` is added above, isn't it enough to filter cells of the `mvp contract`?

Comment (yanguoyu) [reply packages/samples/mvp-dapp/src/actors/record.model.ts:54]: You are right, it's enough.

**Pull request closed without merging**